OpenIPSL/Examples/Solar/PowerFactory/DIgSILENT_PV.mo

```diff
@@ -0,0 +1,25 @@
+within OpenIPSL.Examples.Solar.PowerFactory;
+
+model DIgSILENT_PV
+  extends Modelica.Icons.Example;
+  Modelica.Blocks.Sources.Step voltage(height = -0.05, offset = 1, startTime = 0.5)  annotation(
+    Placement(visible = true, transformation(origin = {70, -20}, extent = {{10, -10}, {-10, 10}}, rotation = 0)));
+  OpenIPSL.Electrical.Solar.PowerFactory.General.ElmVac voltage_source(angle_0 = 0, v_0 = 1)  annotation(
+    Placement(visible = true, transformation(origin = {30, 0}, extent = {{10, -10}, {-10, 10}}, rotation = 0)));
+  Modelica.Blocks.Sources.Step frequency(height = -0.05, offset = 1, startTime = 1) annotation(
+    Placement(visible = true, transformation(origin = {70, 30}, extent = {{10, -10}, {-10, 10}}, rotation = 0)));
+  inner OpenIPSL.Electrical.SystemBase SysData annotation(
+    Placement(visible = true, transformation(origin = {-70.5, 90}, extent = {{-29.5, -10}, {29.5, 10}}, rotation = 0)));
+  OpenIPSL.Electrical.Solar.PowerFactory.DIgSILENT.PV_Plant pv_plant(M_b(displayUnit = "V.A") = 0.5e6, P_0 = 300000, angle_0 = 0, v_0 = 1)  annotation(
```

Comment:
is it V.A or VA here after displayUnit? @tinrabuzin

Reply:
@marcelofcastro That's what OpenModelica picks up from MSL. It was not me. :)

Reply:
cool cool, just wanted to make sure it was not a typo haha

Reply:
It does look like it so I went and checked. The unit abbreviation does not make sense if you ask me but maybe there's something that I don't know...

Reply:
maybe the dot is just to make the multiplication of units explicit. It's a bit weird to me as well but @dietmarw will say that I may have a problem with units in Modelica hahaha Anyway, good that's not a typo!

Reply:
It should be `VA`. I guess some version of OMEdit did screw this up. I changed it now to a more meaningful `MVA`